package-lock.json

```diff
@@ -6989,7 +6989,7 @@
     },
     "csv-stringify": {
       "version": "1.0.4",
-      "resolved": "http://registry.npmjs.org/csv-stringify/-/csv-stringify-1.0.4.tgz",
+      "resolved": "https://registry.npmjs.org/csv-stringify/-/csv-stringify-1.0.4.tgz",
```

Comment:
Should not be modified.

Reply:
Ahhhhhh my bad, it's late, didn't mean to commit that!